challenge-react/src/api/charities.js

```diff
@@ -0,0 +1,10 @@
+import { axiosInstance } from '@/helpers';
+import { API } from '@/constants';
+export const charitiesService = (() => {
```

Comment:
How about we do like this?
```
export const charitiesService = {
  fetchAll: () => axiosInstance.get(API.CHARITIES).then((res) => res?.data),
};
```

Reply:
okay, when API service got an error it's a callback to createAsyncThunk and will rejectWithValue to addCase rejected.
and however,  we can catch data but it's not a callback to rejectWithValue in createAsyncThunk function.

Reply:
Firstly, I didn't get your point. By reading the code, it doesn't make sense to go that way. If it's possible, please explain more about this case. Thanks in advance.